WPMedia/fusion-news-theme-blocks
# TMEDIA-238 - update all blocks to use translation keys

## Description
Update all hard coded language strings to use intl.json files

* Updated the logo fallback alt to be a blank - which is a valid alt value. For assistive technologies a blank alt text denotes the image is for decorative purpose. Based on the blocks logic we have the means to set it correctly from the blocks.json file. If it's not set falling back to a value of "Placeholder logo" provides no value.

## Jira Ticket
- [TMEDIA-238](https://arcpublishing.atlassian.net/browse/TMEDIA-238)

## Acceptance Criteria
* Add new keys to Lokalise and set English value
* Update blocks code to use fusion:intl library to render text strings
* Update locale files to have all supported locale have English defaults

## Test Steps

1. Checkout this branch `git checkout TMEDIA-238-update-all-blocks-to-use-translation-keys`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/ads-block,@wpmedia/alert-bar-block,@wpmedia/article-body-block,@wpmedia/card-list-block,@wpmedia/footer-block,@wpmedia/full-author-bio-block,@wpmedia/global-phrases-block,@wpmedia/header-nav-chain-block,@wpmedia/links-bar-block,@wpmedia/numbered-list-block,@wpmedia/placeholder-image-block,@wpmedia/results-list-block,@wpmedia/search-results-list-block,@wpmedia/share-bar-block,@wpmedia/simple-list-block,@wpmedia/full-author-bio-block,@wpmedia/author-bio-block`
3. Validate English strings are still appearing in none English locale sites - you should not see the language string fallback
4. Sites 
    * EN - http://localhost/pf/all-blocks/?_website=the-gazette
    * FR - http://localhost/pf/all-blocks/?_website=the-sun
    * JA - http://localhost/pf/all-blocks/?_website=arc-demo-3


## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
